Library/Homebrew/test/cask/dsl_spec.rb

```diff
@@ -168,37 +168,37 @@
         cask.config = config
       end
 
-      context "when it is 'zh'" do
+      context "to 'zh'" do
```

Comment:
```suggestion
      describe "to 'zh'" do
```

Sorry, I think I changed my mind on this. Adding `to` to the list of words doesn't really make sense, I think. It doesn't really fit with the rest of the `context` words. I'm basing this partially off of the suggestions [here](https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspeccontextwording). This seems like an exception, so maybe we should just use `describe` here, instead?

Again, though, I'd be fine with merging this as-is. I apologize for changing my mind on this.

Reply:
No problem! I also thought about `describe` for these as I was trying to sleep at about 3am. 😂